__tests__/app.test.js

```diff
@@ -152,20 +152,19 @@ describe("nc-news", () => {
             expect(comment).toHaveProperty("body");
             expect(comment).toHaveProperty("article_id");
           });
-          expect(comment[0].comment_id).toBe(1);
-          expect(comment[0].votes).toBe(1);
-          expect(comment[0].created_at).toBe(1);
-          expect(comment[0].author).toBe(1);
-          expect(comment[0].body).toBe(1);
-          expect(comment[0].article_id).toBe(1);
+          expect(allComments[0].comment_id).toBe(5);
+          expect(allComments[0].votes).toBe(0);
+          expect(allComments[0].author).toBe("icellusedkars");
+          expect(allComments[0].body).toBe("I hate streaming noses");
+          expect(allComments[0].article_id).toBe(1);
         });
```

Comment:
We also want a 200 test for when the ID is valid, but it has no comments - should come back with an empty array then